dev/doc/changes.md

```diff
@@ -1,3 +1,20 @@
+## Changes between Coq 8.8 and Coq 8.9
+
+### ML API
+
+Proof engine
+
+  Note that the core function `EConstr.to_constr` now _enforces_ by
+  default that the resulting term is ground, that is to say, free of
+  Evars. This is usually what you want, open terms should indeed be of
+  type `EConstr.t` as to profit of the invariants the `EConstr` API is
+  meant to provide.
```

Comment:
I suggest "open terms should be of type EConstr.t to benefit from the invariants the EConstr API guarantees"

Reply:
Thanks, incorporated almost all, but kept the "is meant to" as IMHO the softer language matches the actual guarantees better.